### PR TITLE
update readme example

### PR DESCRIPTION
**Pull request by Aprillion**
Update README as discussed in https://github.com/mswjs/msw/issues/254#issuecomment-739507313

* add `id` parameter for constructor
* return the new array from updater function
* auto-formatted with default prettier config => is it OK or do you want add custom prettier or eslint config to the project?

### CODE REVIEW

**Comment by kettanaito** [comment]
Thank you for this improvement, @Aprillion! Don't worry about the formatting, I will introduce Prettier configuration later and re-format the code.

File: README.md [hunk 1]
```diff
@@ -13,15 +13,11 @@ $ npm install live-storage
...
 
-const posts = new LiveStorage([])
+const posts = new LiveStorage("posts", []);
 
-posts.update((prevPosts) => {
-  prevPosts.concat({
-    title: 'Brave new world',
-  })
-})
+posts.update((prevPosts) => [...prevPosts, { title: "Brave new world" }]);
```
Comment (kettanaito) [inline README.md]: Nitpick: could we use `.concat` and just return its result?

```js
return prevPosts.concat({ ... })
```

That's the same logic that with spreading, yet less foreign syntax, so higher recognizability.

**Review by kettanaito — COMMENTED** [review]
I've left one nitpick, please let me know your opinion on it. Thanks!

File: README.md (hunk 1 above)
Comment (Aprillion) [reply README.md]: will do. even if I would think that spread might be more familiar to beginners these days, concat is definitely more searchable, so probably a better choice for a readme

**Review by kettanaito — APPROVED** [review]
Thank you for shipping this, @Aprillion!

**Pull request merged**